DETAILS.md

```diff
@@ -0,0 +1,173 @@
+                  10,166.67   |       0.00  |  10,166.67  |   2,666.67 |    533.33  |    7,500.00 |   3,000.00  |      0.00 |      0.00  |    3,533.33
+         ---------------------+-------------+-------------+------------+------------+-------------+-------------+-----------+------------+---------------
+                 130,000.00*  |       0.00  | 130,000.00* |  32,000.00 |  6,400.00  |   87,000.00 |  38,400.00  |           |            |
+                              |             |             |            |            |  +11,000.00 |  +4,400.00  |           |            |
+                              |             |             |            |            |  =98,000.00 | =39,200.00  |      0.00 |      0.00  |   45,600.00*
+                  10,833.33   |       0.00  |  10,833.33  |   2,666.67 |    533.33  |    8,166.67 |   3,266.67  |      0.00 |      0.00  |    3,800.00
+         ---------------------+-------------+-------------+------------+------------+-------------+-------------+-----------+------------+---------------
+                 150,000.00*  |       0.00  | 150,000.00* |  32,000.00 |  6,400.00  |  107,000.00 |  42,800.00  |           |            |
+                              |             |             |            |            |  +11,000.00 |  +4,400.00  |           |            |
+                              |             |             |            |            | =118,000.00 | =47,200.00  |      0.00 |      0.00  |   53,600.00*
+                  12,500.00   |       0.00  |  12,500.00  |   2,666.67 |    533.33  |    9,833.33 |   3,933.33  |      0.00 |      0.00  |    4,466.67
+         ---------------------+-------------+-------------+------------+------------+-------------+-------------+-----------+------------+---------------
+                 160,000.00*  |       0.00  | 160,000.00* |  32,000.00 |  6,400.00  |  107,000.00 |  42,800.00  |           |            |
+                              |             |             |            |            |  +11,000.00 |  +4,400.00  |           |            |
+                              |             |             |            |            | =118,000.00 | =47,200.00  | 10,000.00 |  4,500.00  |   58,100.00*
+                  13,333.33   |       0.00  |  13,333.33  |   2,666.67 |    533.33  |    9,833.33 |   3,933.33  |    833.33 |    375.00  |    4,841.67
+                              |             |             |            |            |             |             |           |            |
+         ---------------------+-------------+-------------+------------+------------+-------------+-------------+-----------+------------+---------------
+
+         * - past a Gross Salary of £100,000.00, personal allowance rules change - for every extra £2 earned, the personal allowance reduces by £1.
+         Past a salary of £122,000.00, no personal allowance is applicable. And this adjusted excess is taxed at the Higher rate tax.
```

Comment:
I'll remove the word adjusted from master, so please remove it from here as well